nucypher/characters/control/specifications/bob.py

```diff
@@ -42,23 +42,30 @@ class JoinPolicy(BaseSchema):  #TODO:  this doesn't have a cli implementation
 
 class Retrieve(BaseSchema):
     label = fields.Label(
-        required=True, load_only=True,
-        click=options.option_label(required=True))
+        required=True,
+        load_only=True,
+        click=options.option_label(required=False)
```

Comment:
Presumably, there are no side effects of this change, and the only effect is what the PR intended i.e. that Bob will be prompted for the label if not explicitly specified.

Reply:
Yes, that's correct, the Click options need be be optional to allow the fallback to interactive collection.